docs/api.rst

```diff
@@ -153,7 +174,7 @@ Task
     error:: Time -> Error -> void
   }
 
-A Task is any unit of work that can be scheduled for execution on a Scheduler.
+A Task is any unit of work that can be scheduled for execution in a Scheduler.
```

Comment:
"in" still sounds odd to me.  What about "with" or "using"?  I guess there are at least a couple ways to look at a Scheduler: 1) as a container of Tasks, 2) as an actor whose job it is to schedule tasks (and scheduled tasks "reside" in some other container).

2 may be more technically correct, since Timeline is the ultimate container.  If that's the case, then "with" or "using" may be more appropriate.

Thoughts?

Reply:
I agree with 2. I used _in_ because I looked at a Scheduler as a mechanism.